### PR TITLE
sonic-swss-common: Add APP_VXLAN_TUNNEL_TABLE macro to schema

### PR DESCRIPTION
**Pull request by Pterosaur**
* Add APP_VXLAN_TUNNEL_TABLE macro to schema

Signed-off-by: Ze Gan <zegan@microsoft.com>

### CODE REVIEW

File: common/schema.h [hunk 1]
```diff
@@ -37,6 +37,7 @@ namespace swss {
 #define APP_VNET_RT_TABLE_NAME          "VNET_ROUTE_TABLE"
 #define APP_VNET_RT_TUNNEL_TABLE_NAME   "VNET_ROUTE_TUNNEL_TABLE"
 #define APP_VXLAN_VRF_TABLE_NAME        "VXLAN_TUNNEL_MAP"
```
Comment (prsunny) [inline common/schema.h]: How are we handling current Vlan map `CFG_VXLAN_TUNNEL_TABLE_NAME`? I think, we would need to change `APP_VXLAN_VRF_TABLE_NAME` to `APP_VXLAN_TUNNEL_MAP_TABLE_NAME` for generic handling and make changes to `vxlanorch`

**Comment by prsunny** [comment]
Change looks good. Will review along with sonic-swss and complete the merge

**Comment by Pterosaur** [comment]
> and complete

**Comment by Pterosaur** [comment]
> Change looks good. Will review along with sonic-swss and complete the merge

Because my wrong click so that this pull request has been closed
Very apologized for any inconvenience bringing to you.

**Pull request closed without merging**